package.json

```diff
@@ -9,17 +9,16 @@
     "copy:license": "cp ./README.md dist/media-viewer",
     "copy:readme": "cp ./LICENSE dist/media-viewer",
     "copy:docs": "npm run copy:readme && npm run copy:license",
-    "copy:assets": "npm run copy:lib-assets && npm run copy:hmcts-assets && npm run copy:govuk-assets",
+    "copy:assets": "npm run copy:lib-assets",
     "copy:lib-assets": "rsync -r projects/media-viewer/src/assets/* dist/media-viewer/assets/",
-    "copy:hmcts-assets": "rsync -r projects/media-viewer/node_modules/@hmcts/frontend/* dist/media-viewer/assets/hmcts-frontend",
     "copy:govuk-assets": "rsync -r projects/media-viewer/node_modules/govuk-frontend dist/media-viewer/assets",
     "install:js-dependencies": "cd projects/media-viewer && npm install",
     "copy:js-dependencies": "rsync -r projects/media-viewer/node_modules/pdfjs-dist/build dist/media-viewer/assets",
     "copy:files": "npm run copy:js-dependencies && npm run copy:assets && npm run copy:docs",
     "build:lib": "npm install && npm run install:js-dependencies && cd projects/media-viewer && ng build",
     "npm-pack": "cd dist/media-viewer && npm pack",
     "package": "npm run build:lib && npm run copy:files && npm run npm-pack",
-    "test": "cd projects/media-viewer; ng test",
```

Comment:
we do it this way so that the coverage report is generated correctly. did you check whether you get all the files in the coverage report without moving to the projects folder?

Reply:
cd-ing into media-viewer project then not running the command from there but running ng test that is the global command makes no difference where do u run it form